MonitoringSystem/DB/test/Test_monitoringdb.py

```diff
@@ -74,12 +74,13 @@ def test_determineBucketSize( self ):
 
     result = client._determineBucketSize( 1427038370, 1458660770 )
     self.assert_( result['OK'] )
-    self.assertEqual( result['Value'], ( '1w', 86400 * 7 ) )
+    self.assertEqual( result['Value'], ( '7w', 86400 * 7 * 7 ) )
 
 
     result = client._determineBucketSize( 1300807841, 1458660641 )
-    self.assert_( result['Message'] )
-
+    self.assert_( result['OK'] )
```

Comment:
Any idea why on this assertion changed?

Also, `assert_` is now deprecated in favor of `assertTrue()`: https://docs.python.org/2/library/unittest.html#deprecated-aliases